tests/e2e/pages/xpath.page.ts

```diff
@@ -0,0 +1,21 @@
+import { Client, Element, RawResult } from "webdriverio";
+
+import { BasePage } from "./base.page";
+
+class XpathPage extends BasePage {
```

Comment:
XPath

Reply:
No my brother. When its not a sentence but code. Camel case rules ;). Eg we use `Id` in code instead of `ID`